Update TTS task content (#948)
huggingface/hub-docs: tasks/src/text-to-speech/about.md

## Use Cases

> Text-to-Speech (TTS) models can be used in any speech-enabled application that requires converting text to speech imitating human voice.

Reviewer: ```suggestion
Text-to-Speech (TTS) models can be used in any speech-enabled application that requires converting text to human speech in audio form.
```

Author: I just thought about this. This sounds a bit weird. Is it okay if we can ignore this? :)

> Text-to-Speech (TTS) models can be used in any speech-enabled application that requires converting text to speech imitatting human voice.

Reviewer 2: ```suggestion
Text-to-Speech (TTS) models can be used in any speech-enabled application that requires converting text to speech imitating human voice.
```

> ## Inference

Reviewer 3: ```suggestion
## Inference API
```

The Hub contains over [1500 TTS models](https://huggingface.co/models?pipeline_tag=text-to-speech&sort=downloads) that you can use right away by trying out the widgets directly in the browser or calling the models as a service using the Inference API. Here is a simple code snippet to do exactly this:

```python
import json
import requests

headers = {"Authorization": f"Bearer {API_TOKEN}"}
API_URL = "https://api-inference.huggingface.co/models/microsoft/speecht5_tts"

def query(payload):
	response = requests.post(API_URL, headers=headers, json=payload)
	return response

output = query({"text_inputs": "This is a test"})
```

> You can also use libraries such as [espnet](https://huggingface.co/models?library=espnet&pipeline_tag=text-to-speech&sort=downloads) if you want to handle the Inference directly.

Reviewer 3: ```suggestion
You can also use libraries such as [espnet](https://huggingface.co/models?library=espnet&pipeline_tag=text-to-speech&sort=downloads) or [transformers](https://huggingface.co/models?pipeline_tag=text-to-speech&library=transformers&sort=trending) if you want to handle the Inference directly.
```

> 

Reviewer 3: ```suggestion
## Direct Inference
```
